pkg/scheduler/apis/config/v1beta1/defaults_test.go

```diff
@@ -424,6 +424,29 @@ func TestPluginArgsDefaults(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "NodeResourcesBalancedAllocationArgs resources empty",
+			in:   &v1beta1.NodeResourcesBalancedAllocationArgs{},
+			want: &v1beta1.NodeResourcesBalancedAllocationArgs{
+				Resources: []v1beta1.ResourceSpec{
+					{Name: "cpu", Weight: 1},
+					{Name: "memory", Weight: 1},
+				},
+			},
+		},
+		{
+			name: "NodeResourcesBalancedAllocationArgs resources with value",
+			in: &v1beta1.NodeResourcesBalancedAllocationArgs{
+				Resources: []v1beta1.ResourceSpec{
+					{Name: "resource", Weight: 1},
+				},
+			},
+			want: &v1beta1.NodeResourcesBalancedAllocationArgs{
+				Resources: []v1beta1.ResourceSpec{
+					{Name: "resource", Weight: 1},
+				},
+			},
+		},
```

Comment:
Can you add some additional tests:

- resource with a qualified name
- resource with both qualified name and reserved names (cpu/mem)

And intersect them with the case w and w/o specifying weight.

Reply:
sure, will add those testcases in the next iteration.

Reply:
Done